src/components/Table.svelte

```diff
@@ -22,17 +22,11 @@
     let currentRow = undefined
     let extraContent = undefined
 
-    if (e.target.parentNode.classList.contains("title")) {
-      title = e.target.parentNode
-      currentRow = title.nextElementSibling
-      extraContent = e.target.parentNode.nextElementSibling
-    } else {
-      title = e.target.parentNode.parentNode
-      currentRow = title.nextElementSibling
-      extraContent = e.target.parentNode.parentNode.nextElementSibling
-    }
+    title = e.target.parentNode.parentNode
+    currentRow = title.nextElementSibling
+    extraContent = e.target.parentNode.parentNode.nextElementSibling
 
-    title.classList.toggle("title--active")
+    title.classList.toggle("active")
```

Comment:
We need the `--active` for the arrow animation to work when the row is expanded
```suggestion
    title.classList.toggle('title--active')
```
![image](https://github.com/CSIS-iLab/poni-table/assets/51330062/48f90e24-37d3-4da0-b6bc-f5bdcb88836b)

without
![image](https://github.com/CSIS-iLab/poni-table/assets/51330062/ce10d2d0-deae-4cb4-8b4b-f8bfe5f02630)